storybook/stories/wallets/send/WalletSend.stories.tsx

```diff
@@ -376,32 +375,28 @@ storiesOf('Wallets|Send', module)
     };
     return (
       <div>
-        <WalletSendAssetsConfirmationDialog
+        <WalletSendConfirmationDialogView
           amount="20.000000"
+          areTermsAccepted={boolean('areTermsAccepted', true)}
           wallet={wallet}
           totalAmount={new BigNumber('21.000000')}
-          // @ts-ignore ts-migrate(2769) FIXME: No overload matches this call.
-          sender={
-            generateWallet('Wallet name', '45119903750165', walletTokens).id
-          }
           receiver={generateHash()}
           selectedAssets={confirmationTokens}
-          allAvailableTokens={confirmationTokens}
+          assetTokens={confirmationTokens}
           assetsAmounts={confirmationTokensAmounts}
           transactionFee="1.000000"
-          amountToNaturalUnits={formattedAmountToNaturalUnits}
-          onSubmit={() => null}
-          isSubmitting={false}
-          error={null}
-          isFlight={false}
-          onCancel={() => null}
-          onExternalLinkClick={() => null}
           hwDeviceStatus={HwDeviceStatuses.CONNECTING}
+          isFlight={boolean('isFlight', false)}
           isHardwareWallet={boolean('isHardwareWallet', false)}
-          onInitiateTransaction={() => null}
-          onCopyAssetParam={() => {}}
+          isSubmitting={boolean('isSubmitting', false)}
           isTrezor={boolean('isTrezor', false)}
           formattedTotalAmount="21.000000"
+          error={null}
+          onCancel={() => null}
```

Comment:
Wouldn't be better to replace `() => null` by an `action('onCancel')` etc ?

Reply:
Updated, thanks!